titiler/core/setup.py

```diff
@@ -6,15 +6,14 @@
     long_description = f.read()
 
 inst_reqs = [
-    "fastapi==0.63.0",
+    "fastapi==0.65.1",
```

Comment:
do we need to pin fastapi ?

Reply:
IMO `titiler.core` shouldn't strictly pin fastapi. It's still worthwhile to provide a range if you know you use specific features, but I think the _application_ should be the one strictly pinning dependencies.